### DISCUSSION TITLE
Silencing warning about no backend having been added in i18next.use()

### DISCUSSION THREAD

**Discussion by raine** [i18next/i18next#1669] [Q&A] [answered]
I was wondering about this warning in the console:

```
i18next.js:28 i18next::backendConnector: No backend was added via i18next.use. Will not load resources.
```

After some research, it looks like that in our project, `i18next` is initialized without a backend, and translations are added using `i18n.addResourceBundle()`. So I believe the warning does not seem to apply in this case.

To reduce future confusion and noise in console, I'd like to silence that warning. Any ideas on how to accomplish that?

Thank you.

**Comment by jamuhl** [comment, marked as answer]
init with options `resources: {}` https://www.i18next.com/how-to/add-or-load-translations#add-on-init

if not adding resources (even if just empty) i18next will assume you're using a backend

**Reply by raine** [reply]
Thank you!
